envConfig/live.env

```diff
@@ -8,3 +8,4 @@ SIMORGH_TOGGLES_URL=https://toggles.api.bbci.co.uk
 LOG_DIR='/var/log/simorgh'
 LOG_LEVEL='info'
 SIMORGH_APP_ENV=live
+SIMORGH_INCLUDES_ENDPOINT=https://www.bbc.com/ws/includes
```

Comment:
Could we call this something like `SIMORGH_INCLUDES_BASE_URL` for consistency with the above`, and move this line up to be with the other URLs so it doesn't get missed?

Reply:
@jamesdonoh done!